docs: add Getting Started section (#34)
NordicSemiconductor/NAT-TestFirmware: README.md

[![Nordic ClangFormat](https://img.shields.io/static/v1?label=Nordic&message=ClangFormat&labelColor=00A9CE&color=337ab7)](https://github.com/nrfconnect/sdk-nrf/blob/master/.clang-format)
[![Zephyr compliance](https://img.shields.io/static/v1?label=Zephry&message=compliance&labelColor=4e109e&color=337ab7)](https://docs.zephyrproject.org/latest/contribute/index.html#coding-style)

> It determines NAT timeouts in a cellular networks by sending messages to the [NAT-TestServer](https://github.com/NordicSemiconductor/NAT-TestServer) and waiting for the reply.

Reviewer: ```suggestion
This firmware determines the NAT timeouts in cellular networks by sending messages to the [NAT-TestServer](https://github.com/NordicSemiconductor/NAT-TestServer) and waiting for the reply.
```

This application is built with [sdk-nrf](https://github.com/nrfconnect/sdk-nrf).

## Getting started

> Please help us learn more about NAT timeouts in cellular networks:

Reviewer: ```suggestion
Support this project by giving more insight into the NAT timeouts in cellular networks.
The following steps describe the testing process:
```

1. Find the latest firmware build in the [releases](https://github.com/NordicSemiconductor/NAT-TestFirmware/releases) and flash it onto your nRF9160 Development Kit.

> 1. Put in the SIM card of your choice and power on the kit. The test will start automatically. **Do not change the location of the kit during testing, switching mobile cells must be avoided.**

Reviewer: ```suggestion
1. Insert the SIM card of your choice and power on the development kit. The test starts automatically. **Do not change the location of the development kit during testing, and avoid switching mobile cells.**
```

> 1. Optionally: you may connect it via USB to observe the test runs in a terminal.

Reviewer: ```suggestion
1. Optionally, you can connect the development kit via USB and observe the test status in a terminal.
```

> 1. Wait until the test finish (indicated by the 4 LEDs blinking in a rotating pattern). If the TCP test is still running after 24 hours, you can abort it. We generally assume that more than 24 hours means sufficient power savings are provided from the network for the majority of use case scenarios.

Reviewer: ```suggestion
1. Wait until the test finishes (This is indicated by the 4 LEDs, blinking in a rotating pattern). If the TCP test continues to run for more than 24 hours, you can abort it. This firmware assumes that a test duration exceeding 24 hours indicates a network providing sufficient power savings for majority of the use case scenarios.
```

Author: > This firmware assumes that a test duration exceeding 24 hours indicates a network providing sufficient power savings for majority of the use case scenarios.

No, that is not a feature of the firmware. It will keep testing forever. There is no upper time limit.

Reviewer: ```suggestion
1. Wait until the test finishes (This is indicated by the 4 LEDs, blinking in a rotating pattern). If the TCP test continues to run for more than 24 hours, you can abort it. It is generally assumed that a test run duration exceeding 24 hours indicates a network providing sufficient power savings for majority of the use case scenarios.
```

> 1. Register an account on <https://cellprobe.thingy.rocks/> and log-in to see your test-results (they are updated every hour).

Reviewer: ```suggestion
1. Register an account on <https://cellprobe.thingy.rocks/> and login to see your test results (These results are updated every hour).
```

> 1. If your SIM does not show up this could mean that the ICCID is unknown. Please open an issue in the [TestServer repository](https://github.com/NordicSemiconductor/NAT-TestServer/issues/new).

Reviewer: ```suggestion
1. If your SIM does not show up after login, it could mean that the ICCID is unknown. In this case, open an issue in the [TestServer repository](https://github.com/NordicSemiconductor/NAT-TestServer/issues/new).
```

> 1. Optionally: repeat from Step 2 for every SIM you would like to test.

Reviewer: ```suggestion
1. Optionally, repeat the steps from Step 2 for every SIM you would like to test.
```

> Questions? Please open an issue [in the TestServer repository](https://github.com/NordicSemiconductor/NAT-TestServer/issues/new).

Reviewer: ```suggestion
If you have any questions, open an issue [in the TestServer repository](https://github.com/NordicSemiconductor/NAT-TestServer/issues/new).
```
